src/Rings/mpolyquo-localizations.jl

```diff
@@ -1398,6 +1406,10 @@ function saturated_ideal(I::MPolyQuoLocalizedIdeal)
   return saturated_ideal(pre_image_ideal(I))
 end
 
+# for convenience of the scripting user allow I::MPolyQuoIdeal as 
+# input to saturated_ideal and return the preimage of I in 
+# under the canonical quotient map 
```

Comment:
```suggestion
# We currently support rings of type `MPolyRing`, `MPolyQuo`, `MPolyLocalizedRing`, and `MPolyQuoLocalizedRing` 
# as internals for the affine schemes. All of these rings `R` have a polynomial ring `P` in the background with a 
# canonical morphism `can : P -> R`. Since this polynomial ring `P` is usually relevant for computations, we would like 
# to provide the user with coherent methods to obtain both the ring `P` itself (via `base_ring`) and also the preimage 
# of ideals `I` in `R` under `can`. For an ideal in a localized ring this command is `saturated_ideal(I)` and we extend 
# that command to also return coherent results for the other types of rings. 
```

Reply:
I do not yet see a version for `saturated_ideal` for input of type `MPolyIdeal`. Am I missing something?